src/Whim/Resources/Defaults.xaml

```diff
@@ -0,0 +1,72 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+	<!--  Bar height. Must not contain any property other than "Height"  -->
+	<Style x:Key="bar:height" TargetType="RelativePanel">
+		<Setter Property="Height" Value="30" />
```

Comment:
On my PC, the default `ActualHeight`  is 39, though this could be a peculiarity of my system.

Reply:
> Do you mean without this style or with this style? This style effectively just sets `BarConfig._height`, so all the dpi-related scaling applied to it, would also affect the actual height.

With this style. What was actually rendered was a 39px bar. This caused the bar to clip the windows a bit:

![image](https://github.com/dalyIsaac/Whim/assets/17995621/8730ac05-ba7b-4c92-8732-1b44d0b9c2b6)

This seems to be scaling related - the bar is exactly the right size with `30px` and 150% scaling:

<img width="251" alt="image" src="https://github.com/dalyIsaac/Whim/assets/17995621/e039dc5c-fdc7-442e-aaaf-5364f5292a61">

> We can also postpone to the follow up on additional themes and tuning the defaults.

Let's do this.